classes/MockDisablerPHPUnit10.php

```diff
@@ -22,15 +22,22 @@ class MockDisablerPHPUnit10 implements FinishedSubscriber
      * @var Deactivatable The function mocks.
      */
     private $deactivatable;
+
+    /**
+     * @var callable|null The callback to execute after the test.
+     */
+    private $callback;
     
     /**
      * Sets the function mocks.
      *
      * @param Deactivatable $deactivatable The function mocks.
+     * @param callback|null $callback      The callback to execute after the test.
      */
-    public function __construct(Deactivatable $deactivatable)
+    public function __construct(Deactivatable $deactivatable, ?callable $callback = null)
```

Comment:
```suggestion
    public function __construct(Deactivatable $deactivatable, ?\Closure $callback = null)
```

the reason is to not allow array callable